### DISCUSSION TITLE
Hardware requirement of private, local network, with few user case

### DISCUSSION THREAD

**Discussion by kuchienkz** [mediacms-io/mediacms#1006] [Q&A]
I'm planning to use MediaCMS to manage and watch my movies, accessing it from different devices, mostly with local network. The user probably just me and my family (up to 4 person). The movies varies in sizes, from 480p to 4K.

I'm planning to run it on my existing [Intel N100](https://www.intel.com/content/www/us/en/products/sku/231803/intel-processor-n100-6m-cache-up-to-3-40-ghz/specifications.html) system.

Now for the questions:
1. Can I use MediaCMS **comfortably** with Intel N100? Note that I'm also running another tasks with the system.
2. How much RAM?
3. Since I use it with local network anyway, is there a way to disable the transcoding?

**Comment by AnthonyTorreso** [comment]
Good Morning,

   For a local/home/intranet site with only a handful of people I personally don't see a reason that your N100 would not work. 
As far as ram, get at least 8GB's as the price is pretty much nothing to go from 4G->8G.  Now please note this will depend on what else your using the box for, if it just movies then I don't see why this wouldn't be fine.

Now for your disabling the transcoding. This is now not a problem, at the end of the settings.py file is the following line.
DO_NOT_TRANSCODE_VIDEO = False

Change the False to True, then restart your instance via systemctl restart mediacms celery_long celery_short 
and you should be good to go.

Thanks so much,
Anthony
